tm2/pkg/bft/rpc/client/rpc_test.go

```diff
@@ -384,24 +381,21 @@ func TestTx(t *testing.T) {
 	cases := []struct {
 		valid bool
 		hash  []byte
-		prove bool
 	}{
 		// only valid if correct hash provided
-		{true, txHash, false},
-		{true, txHash, true},
-		{false, anotherTxHash, false},
-		{false, anotherTxHash, true},
-		{false, nil, false},
-		{false, nil, true},
+		{true, txHash},
+		{true, txHash},
+		{false, anotherTxHash},
+		{false, anotherTxHash},
+		{false, nil},
+		{false, nil},
```

Comment:
I think there are some duplicate tests case leftovers 😬

Reply:
I am very reluctant to tinker with legacy tests. I've cleaned it up, just a bit:

[382e65c](https://github.com/gnolang/gno/pull/1546/commits/382e65c9a644d01a50f7e7b8e5f80bfda8d71a74)